README.md

```diff
@@ -1,4 +1,53 @@
-# devtools-protocol
+# DevTools protocol
 
-This is the repository for devtools-protocol. You're welcome to
-[[contribute]](CONTRIBUTING.md)!
+This repository is for resources and discussion around aligning the developer tools wire protocol used by various browsers, JavaScript egines and debugging tools.
```

Comment:
Typo: engines

Reply:
Do we include in "wire protocol" the tech stack underlying the protocol, like REST endpoints, WebSockets, and JSON-RPC?

Reply:
Sure, how about I just say generically "protocol"?